docs/content/users/extend/custom-commands.md

```diff
@@ -36,11 +36,11 @@ open -a PhpStorm.app ${DDEV_APPROOT}
 
 To provide a command which will execute in a container, add a Bash script to `.ddev/commands/<container_name>`, for example, `.ddev/commands/web/mycommand`. The Bash script will be executed inside the named container. For example, see the [several standard DDEV script-based web container commands](https://github.com/ddev/ddev/blob/master/pkg/ddevapp/global_dotddev_assets/commands/web).
 
-You can run commands in custom containers as well as standard DDEV `web` and `db` containers. Use the service name, like `.ddev/commands/solr/<command>`. The only catch with a custom container is that your service must mount `/mnt/ddev_config` like the `web` and `db` containers do; the `volumes` section of `docker-compose.<servicename>.yaml` needs:
+You can run commands in custom containers as well as standard DDEV `web` and `db` containers. Use the service name, like `.ddev/commands/solr/<command>`. The only catch with a custom container is that your service must mount `/mnt/ddev-global-cache` like the `web` and `db` containers do; the `volumes` section of `docker-compose.<servicename>.yaml` needs:
```

Comment:
Bear in mind that this page isn't about how to make custom extensions or docker containers etc. See [docs/content/users/extend/custom-compose-files.md](https://github.com/ddev/ddev/pull/5753/files#diff-0f22c880ae6590eb0f249987709640c16ceb7cba5946361032b7328e4a3f665b) which does still mention both.

If you want to mention `/mnt/ddev_config` here the docs should probably say how that ties into custom commands, which after this change I don't think it will, directly? Maybe it's just better to not mention either mount directly and instead link to the page that does?